openshift/wps-api-base/openshift/build.yaml

```diff
@@ -1,25 +1,25 @@
-apiVersion: v1
+apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
   creationTimestamp: null
-  name: uvicorn-gunicorn-fastapi
+  name: wps-api-base
 labels:
   app: ${NAME}${SUFFIX}
   phase: build
-  app.kubernetes.io/name: uvicorn-gunicorn-fastapi
+  app.kubernetes.io/name: wps-api-base
   app.kubernetes.io/managed-by: template
-  app.kubernetes.io/version: "python3.8"
+  app.kubernetes.io/version: "ubuntu.22.04"
 parameters:
   - name: NAME
     value: wps-api-base
   - name: SUFFIX
   - name: VERSION
     description: Output version
-    value: "python3.9-latest"
+    value: "ubuntu.22.04-latest"
   - name: GIT_URL
     value: https://github.com/bcgov/wps.git
   - name: GIT_BRANCH
-    value: main
+    value: gunicorn_python
```

Comment:
This will need to be changed back to main at some point, right?

Reply:
Yep - I have to remember to do it before I merge.

Reply:
Reverted.